types/atom/index.d.ts

```diff
@@ -199,6 +202,18 @@ export interface AtomEnvironment {
     /** Toggle the full screen state of the current window. */
     toggleFullScreen(): void;
 
+    /**
+     * Restores the full screen and maximized state after the window has resized to prevent resize
+     * glitches.
+     */
+    displayWindow(): void;
+
+    /** Get the dimensions of this window. */
+    getWindowDimensions(): { x: number, y: number, width: number, height: number };
+
+    /** Set the dimensions of the window. */
+    setWindowDimensions(dimensions: object): Promise<object>;
```

Comment:
Going by the [source code](https://github.com/atom/atom/blob/v1.36.0/src/atom-environment.js#L704), we can define the type of this parameter as: `{ x?: number, y?: number, height?: number, width?: number }`.